homeassistant/components/ozw/light.py

```diff
@@ -118,6 +209,38 @@ async def async_turn_on(self, **kwargs):
         """Turn the device on."""
         self.async_set_duration(**kwargs)
 
+        rgbw = None
+        white = None
+        hs_color = None
+
+        if kwargs.get(ATTR_WHITE_VALUE) is not None:
+            white = kwargs[ATTR_WHITE_VALUE]
+
+        if kwargs.get(ATTR_COLOR_TEMP) is not None:
+            rgbw = "#00000000ff"
```

Comment:
Why do we always send white RGBW when getting a color temp value?

Reply:
Good question, since I wasn't sure how the color_temp should work I just left what was already there.

Reply:
How do you mean "left what was already there"?

The ozw light platform didn't support color temperature before, right? If we don't know how to support it, we shouldn't support it.

Reply:
> How do you mean "left what was already there"?

It's how it was in the zwave platform.

> The ozw light platform didn't support color temperature before, right? If we don't know how to support it, we shouldn't support it.

So nuke it and come back to it later?